app/inbound/calculation/process-calculation.js

```diff
@@ -0,0 +1,30 @@
+const db = require('../../data')
+
+const savePlaceholderOrganisation = require('./save-placeholder-organisation')
+const getCalculationBySbi = require('./get-calculation-by-sbi')
+const saveCalculation = require('./save-calculation')
+const saveFundings = require('./save-fundings')
+const updateCalculation = require('./update-calculation')
+
+const processCalculation = async (calculation) => {
+  const transaction = await db.sequelize.transaction()
+
+  try {
+    const existingCalculation = await getCalculationBySbi(calculation.sbi, transaction)
```

Comment:
An existing calculation should be `calculationRef` not SBI.  One SBI will have multiple legitimate calculations if they've had a post payment adjustment.

Reply:
I hadda comment next to this asking tht exact ques on which field to use